scripts/installDependencies-macos.sh

```diff
@@ -29,7 +29,7 @@ if [ -z "$PROTOBUF" ]; then
     mkdir -p build && cd build &&
     # use cmake instead of autogen.sh so that protobuf-config.cmake can be installed
     cmake -Dprotobuf_BUILD_TESTS=OFF -DCMAKE_BUILD_TYPE=Debug .. &&
-    make && make install && make clean && ldconfig
+    make && make install && make clean
```

Comment:
you can leave this change to https://github.com/eBay/Gringofts/pull/60 or you have to resolve conficts

Reply:
i'll leave all macos-related changes to you. please make sure it compiles.